tests/Integration/ValidationTest.php

```diff
@@ -427,14 +427,70 @@ public function itSetsArgumentsOnCustomValidationDirective(): void
 
         $this->assertSame(
             [
-                'input.name' => [
+                'name' => [
```

Comment:
Optimally, the error would still contain the original path to the argument.

Reply:
This is because the tests uses `@spread`

Reply:
Right, i just noticed that. We don't have to fix it in this PR.